templates/github/.github/workflows/publish_docs.yml.j2

```diff
@@ -0,0 +1,68 @@
+{% include 'header.j2' %}
+---
+name: Publish docs
+
+on:
+{% if publish_docs_to_pulpprojectdotorg %}
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+{% endif %}
```

Comment:
This doesn't work well. It raises an error: `No event triggers defined in 'on'`.

Reply:
Is it sufficient to just let this happen nightly?

Reply:
You mean the workflow or the error?

Reply:
The workflow

Reply:
I don't have a strong opinion here and can change it to only run nightly. Any objections @pulp/continuous-integration?

Reply:
no objections

Reply:
I find it useful being able to see the built docs right away. What is the concern about the frequency?

Reply:
My main reasons were

* Releases already trigger a documentation publish which made the "tags" trigger redundant
* If we want to run it any time a branch gets pushed, it doesn't need to be a separate workflow, we can just add it to the normal "ci" workflow and avoid setting up Pulp all over again
* It feels wasteful to have a totally separate job for it, and we should probably try to avoid going crazy with our CI consumption in the future

Reply:
I filed an issue here https://pulp.plan.io/issues/7910 to continue the conversation. I'm going to merge this so we can start on the pulp_file 1.4.0 release and then revisit later.